bouncer/shared/setup_boost_pools.ts

```diff
@@ -1,19 +1,78 @@
-import { ingressEgressPalletForChain, getAssetsForChain, Asset } from '../shared/utils';
+import { InternalAssets as Assets } from '@chainflip/cli';
+import {
+  ingressEgressPalletForChain,
+  getAssetsForChain,
+  chainFromAsset,
+  Asset,
+} from '../shared/utils';
 import { submitGovernanceExtrinsic } from '../shared/cf_governance';
 import { observeEvent } from './utils/substrate';
+import { addBoostFunds } from './boost';
 
-type BoostPoolId = {
+export type BoostPoolId = {
   asset: Asset;
   tier: number;
 };
 
 // These are the tiers of boost pools that will be created for each asset
 const boostPoolTiers = [5, 10, 30];
 const chains = ['Ethereum', 'Polkadot', 'Bitcoin', 'Arbitrum'] as const;
```

Comment:
We have a `chainConstants` in the chainflip-cli - perhaps we can use that? Seems hard to believe this is the first time we've required such a constant